<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HProdotti</class>
 <widget class="QWidget" name="HProdotti">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prodotti</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Resources/Wood-4-icon.png</normaloff>:/Resources/Wood-4-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Tipo</string>
             </property>
             <property name="buddy">
              <cstring>cbxType</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cbxType"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="lvMaster">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Cerca</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lnSearch"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>DETTAGLI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lid">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Descrizione</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="desc"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxtipo"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Unit√† di misura</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxUM"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Allergenico</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cballergenico"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Nuovo</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Plus.PNG</normaloff>:/Plus.PNG</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Salva</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Download.PNG</normaloff>:/Download.PNG</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Immagine</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>seleziona immagine</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Map.PNG</normaloff>:/Map.PNG</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Salva immagine</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Download.PNG</normaloff>:/Download.PNG</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Chiudi</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Banned-Sign.PNG</normaloff>:/Banned-Sign.PNG</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
